features/fixtures/TestApp/config/config.yml

```diff
@@ -36,7 +39,26 @@ dunglas_api:
                 default:               3
                 client_can_change:     true
 
```

Comment:
missing `enable_fos_user: true` in the `dunglas_api` configuration.
That's why your listener is never called I guess.